src/iop/colorbalancergb.c

```diff
@@ -2015,25 +2015,37 @@ void gui_init(dt_iop_module_t *self)
   gtk_box_pack_start(GTK_BOX(self->widget), dt_ui_section_label_new(C_("section", "perceptual brilliance grading")), FALSE, FALSE, 0);
   sect = DT_IOP_SECTION_FOR_PARAMS(self, N_("brilliance"));
 
+  /*
+  We really should soft-limit the brilliance sliders to ±20%, as values higher than
+  +20% could quickly lead to heavy artifacts when combined with some other modules.
+  */
   g->brilliance_global = dt_bauhaus_slider_from_params(self, "brilliance_global");
+  dt_bauhaus_slider_set_soft_range(g->brilliance_global, -0.2f, 0.2f);
   dt_bauhaus_slider_set_digits(g->brilliance_global, 4);
   dt_bauhaus_slider_set_format(g->brilliance_global, "%");
-  gtk_widget_set_tooltip_text(g->brilliance_global, _("add or remove brilliance by an absolute amount"));
+  gtk_widget_set_tooltip_text(g->brilliance_global, _("add or remove brilliance by an absolute amount\n"
+                                                      "if the sum of the global brilliance and highlights brilliance values > 20%, this\n"
```

Comment:
A rewording proposal avoiding the "this can cause" after the if, sounds _lighter_ to me:

> if the sum of the global brilliance and highlights brilliance values > 20%, artifacts with any subsequent wavelets, guided filter or blurring operations artifacts may appear.

Reply:
Agreed, it's easier to read. But it seems to me that the second word "artifact" in the sentence can be omitted, no?